app/bolt_listeners.py

```diff
@@ -549,11 +549,8 @@ def ack_summarize_options_modal_submission(
     ack: Ack,
     payload: dict,
 ):
-    where_to_display = (
-        extract_state_value(payload, "where-to-share-summary")
-        .get("selected_option")
-        .get("value", "modal")
-    )
+    selected_option = extract_state_value(payload, "where-to-share-summary").get("selected_option") or {}
```

Comment:
I do understand this change may be necessary to make pytype happy. However, the selected_option must exist in this scenario, so I don't want to have fallback like `or {}` here. Instead, replacing `get("selected_option")` with `["selected_option"]` could make more sense.

Reply:
Thank you for your review. I have corrected the issues you mentioned and made improvements to other parts as well.